python/ray/util/accelerators/accelerators.py

```diff
@@ -13,5 +13,6 @@
 GOOGLE_TPU_V3 = "TPU-V3"
 GOOGLE_TPU_V4 = "TPU-V4"
 
-# Deprecated as A100 is not part of the "TESLA" suite.
-NVIDIA_TESLA_A100 = "A100"
+# Use these instead of NVIDIA_A100 if you need a specific accelerator size.
+NVIDIA_A100_40G = "A100-40G"
+NVIDIA_A100_80G = "A100-80G"
```

Comment:
We need to update the auto-detection logic so that we set the right accelerator type but it will be backward incompatible.

Reply:
Can you point me to the code where this is done?

Reply:
Inside `nvidia_gpu.py`: `get_current_node_accelerator_type()`

Reply:
Oh hm, is there a reference doc or an A100 node I could get my hands on to see what the nvidia GPU name looks like on real hardware? Unfortunately they seem pretty scarce.

Reply:
Actually, I'll just mark these as manual-add for now.